q2_types/per_sample_sequences/_transformer.py

```diff
@@ -94,6 +95,17 @@ def _11(dirfmt: CasavaOneEightLanelessPerSampleDirFmt) \
         dirfmt, SingleLanePerSamplePairedEndFastqDirFmt, parse_lane=False)
 
 
+@plugin.register_transformer
+def _12(dirfmt: SingleLanePerSampleSingleEndFastqDirFmt) \
+        -> CasavaOneEightSingleLanePerSampleDirFmt:
+    casava = CasavaOneEightSingleLanePerSampleDirFmt(mode='w')
+    for file in dirfmt.path.iterdir():
+        if 'MANIFEST' not in str(file) and 'metadata' not in str(file):
+            shutil.copy(os.path.join(str(dirfmt), file), str(casava))
```

Comment:
DBC: sensing a theme yet? Since you are working with `pathlib` objects here, it might be easier to use the pathlib API.